tools/codegen/api/native.py

```diff
@@ -29,10 +29,16 @@ def name(func: FunctionSchema) -> str:
 
 def argumenttype_type(t: Type, *, mutable: bool, binds: ArgName) -> CType:
     if str(t) == 'Tensor?':
```

Comment:
nit: I'd factor out `tensor_type = BaseCType('Tensor', binds)` for clarity, if you're in here again before landing